server/app/views/admin/migration/AdminImportViewPartial.java

```diff
@@ -286,14 +286,11 @@ private DomContent renderQuestion(
             .with(
                 newOrDuplicateIndicator,
                 div()
-                    .with(
-                        TextFormatter.formatText(
-                            question.getQuestionText().getDefault(), false, false))
+                    .with(TextFormatter.formatText(question.getQuestionText().getDefault()))
```

Comment:
I refactored a couple of places on the admin side to use the simpler `formatText` method.